README.md

```diff
@@ -44,11 +45,43 @@ If you are upgrading from a previous version of blender2ogre, and having problem
 * [Animations](http://www.youtube.com/watch?feature=player_embedded&v=5oVM0Lmeb68)
 * [Meshmoon: Video and text instructions how to install and use blender2ogre addon](http://doc.meshmoon.com/index.html?page=from-blender-to-meshmoon-part-1)
 
-## Importing meshes
-As of `blender2ogre` version *0.7.4*, the Kenshi Importer has been integrated into `blender2ogre` with the following features:
+## Exporting Meshes
+To export a blender model: `File Menu > Export > Ogre3D (.scene & .mesh)`. If the menu button is greyed out, the select the object to export from the blender Node tree (Scene collections) first. 
+
+- If you have `OGRETOOLS_XML_CONVERTER` set to a "OgreXMLConverter.exe" path, then the export dialogue will display options relevant for the Ogre (v1) mesh format.
+- If you have `OGRETOOLS_XML_CONVERTER` set to a "OgreMeshTool.exe" path, then the export dialogue will display options relevant for the OgreNext (v2) mesh format. 
+
+### Mesh triangulation issues
+![Cube with broken shading](images/triangulate/broken-shading.png)
+
+If you are seeing some issues with the mesh triangulation (like visible triangles in your shading), check the [Mesh Triangulation README](MeshTriangulation.md) to learn more about the subject and how to workaround it.
+
+### OgreNext tips
+If you do want to export in the OgreNext (v2.) format, make sure in the `Export dialogue > General Settings > Mesh Export Version` is set to V2. The following parameters are a good start point to get a model exported to an Ogre mesh:
+* General
+  - Mesh export version: v2
+* Materials
+  - Export materials: ticked
+* Armature
+  - Armature animation: ticked
+* Mesh
+  - Export mesh: ticked
+  - Edge lists: un-ticked
+  - If your model's materials contain normal mapping:
+    - Tangents: "generate with parity"
+    - Else Tangents: "none"
+  - Optimise Vertex buffers for shaders: ticked
+  - Vertex buffer options: **puqs**
+
+You can check the arguments passed to `OgreMeshTool.exe` in the Blender console. (`Window Menu > Toggle System Console`)
+
+Blender will export the material format in a Ogre (V1) format. This is not compatible with OgreNext (V2.*). You should manually convert them to a material.json file. See the [Ogre Wiki: HLMS Materials](https://wiki.ogre3d.org/HLMS+Materials) for more information.
+
+## Importing Meshes
+As of `blender2ogre` version *0.8.2*, the Kenshi Importer has been integrated into `blender2ogre` with the following features:
  - Import mesh from `.xml` as well as `.mesh` files
  - Option to be able to merge imported submeshes or keep them separate
- - Parsing/Conversion of materials into Blender (not great, but most matching properties have been considered)
+ - Parsing/Conversion of materials into Blender (just create a Principled BSDF material and add a texture with the proper UV mapping)
```

Comment:
I would suggest to leave the README as is. "Principled BSDF" is a Blender 2.8+ only thing

Reply:
also the screenshots are all from blender 2.8, which probably confuse more than they help